monitoring/grafana/dashboards/CMakeLists.txt

```
@@ -12,7 +12,7 @@ endif()
 
 if(WITH_GRAFANA)
   include(AddCephTest)
-  add_tox_test(grafana TOX_ENVS grafonnet-check)
+  add_tox_test(grafana TOX_ENVS grafonnet-check promql-query-test lint)
```

This will like be not only faster (given every ctest thread can now run 3 parallel tests instead of a long one), but in case of error it'll dump only the output of the failed test, and not everything else:
```suggestion
  add_tox_test(grafana TOX_ENVS grafonnet-check)
  add_tox_test(grafana TOX_ENVS promql-query-test)  
  add_tox_test(grafana TOX_ENVS grafonnet-check lint)
```

---

good to know, thanks!

---

update: that doesn't work